init.lua

```diff
@@ -1,13 +1,22 @@
 require('vis')
 
 local function set_title(title)
-	vis:command(string.format(":!printf '\\033]2;vis %s\\007'", title))
+	io.write('\27]2;vis '..title..'\7')
+	io.flush()
```

Comment:
please avoid doing anything with this for this PR, and let's focus on restoring the title

Reply:
> why can't print be used here

you asked for it? okay??